### ISSUE TITLE
Dataplane install wizard suggests the `--mesh` flag when it shouldn't

### ISSUE THREAD

**Issue by jpeach** [kumahq/kuma-gui#169]
The dataplane install wizard generates this `kuma-dp` commandline:
```bash
kuma-dp run \
      --name=example-g68sex \
      --mesh=default \
      --cp-address=https://127.0.0.1:5678/ \
      --dataplane="type: Dataplane
mesh: default
name: example-g68sex
networking: 
  address: 192.168.1.100
  inbound: 
    - port: 80
      servicePort: 6060
      serviceAddress: 127.0.0.1
      tags: 
        kuma.io/service: example
        kuma.io/protocol: tcp" \
      --dataplane-token-file=kuma-token-example-g68sex
```

But `kuma-dp` doesn't allow `--mesh` if it is also set in the dataplane resource. The wizard should omit the flag in this case.

**Comment by tomaszwylezek** [comment]
Hi @jpeach!
I took a look at it and you are right. Also, it looks like it shouldn't produce flag `--name` as the result of the process. I will create a PR that is resolving both of these issues. Great that you found and shared that issue!

**Comment by tomaszwylezek** [comment]
Closing, as it was fixed in #173

**Comment by jpeach** [comment]
Thanks! FYI, you can reference the issue in your commit comments to link everything https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
